dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/presenter/impl/BasePresenterImpl.java

```diff
@@ -46,8 +46,6 @@ protected void bindBaseElements() {
         view.getHeaderViewAboutLink().setOnAction(arg0 -> view.showAboutPopup());
         
         view.getHeaderViewHelpLink().setOnAction(arg0 -> view.showHelpPopup());
-        
-        view.getCancelLink().setOnAction(arg0 -> getController().goToPreviousPage());

```

Comment:
Yes, it would have been, however, the reason this was done is not because of a null check, rather it's different functionality for the back link in different pages, currently only one of course, the Package Metadata page doesn't just go back, it shows the user a warning popup before going back. I couldn't figure out a way to override the action for just that page, so this was the next best thing. You're more than welcome to fix it if you can, I agree it's not ideal.